src/Datadog.Trace.ClrProfiler.Native/clr_helpers.h

```diff
@@ -329,11 +329,12 @@ mdAssemblyRef FindAssemblyRef(
 // disabled_integration_names
 std::vector<Integration> FilterIntegrationsByName(
     const std::vector<Integration>& integrations,
-    const std::vector<WSTRING>& integration_names);
+    const std::vector<WSTRING>& disabled_integration_names);
 
 // FlattenIntegrations flattens integrations to per method structures
 std::vector<IntegrationMethod> FlattenIntegrations(
-    const std::vector<Integration>& integrations);
+    const std::vector<Integration>& integrations,
```

Comment:
This change broke the tests. Fix the compiler error.

Reply:
I went added `FilterIntegrationsByTargetAssemblyName()` instead and reverted this change.